script/start-foreman

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+./node_modules/.bin/webpack-dev-server --config config/webpack.config.js $WEBPACK_OPTS &
+./bin/rails server -b 0.0.0.0 $*
```

Comment:
`"$@"` should be preferred to pass additional args in bash: it handles "" more nicely